components/fieldgroup/stories/template.js

```diff
@@ -46,6 +50,7 @@ export const Template = (
 						label,
 						isRequired,
 						alignment: labelPosition === "side" ? "right" : "top",
+						id: groupLabelId,
 					},
 					context,
 				),
```

Comment:
For the input loop, consider checking one of them to be able to test for sure the immutability. [Here's how I did it](https://github.com/adobe/spectrum-css/pull/3350/files#diff-9d5d7da10a73aa5f35dcd706223f6b2198005d038d0310251ee6d2fb9c673541R72) for the radio update as one option.

Reply:
That's a good idea to show one checked in the example. I added a commit for this. I adjusted it slightly so it won't affect any `isChecked` value coming from the `items` data (in case that changes to show any checked by default): `...(isReadOnly ? {isChecked: i === 1} : {}),`